packages/@aws-cdk/aws-servicecatalog/README.md

```diff
@@ -190,21 +189,21 @@ const product = new servicecatalog.CloudFormationProduct(this, 'MyFirstProduct',
 
 ### Adding a product to a portfolio
 
-You add products to a portfolio to manage your resources at scale.  After adding a product to a portfolio,
-it creates a portfolio-product association, and will become visible from the portfolio side in both the console and service catalog CLI.
-A product can be added to multiple portfolios depending on your resource and organizational needs.
+You add products to a portfolio to organize and distribute your catalog at scale.  Adding a product to a portfolio creates an association,
+and the product will become visible within the portfolio side in both the Service Catalog console and AWS CLI.
+You can add a product to multiple portfolios depending on your organizational structure and how you would like to group access to products.
 
 ```ts fixture=portfolio-product
 portfolio.addProduct(product);
 ```
 
 ## Tag Options
 
-TagOptions allow administrators to easily manage tags on provisioned products by creating a selection of tags for end users to choose from.
-TagOptions are created by specifying a tag key with a selection of allowed values and can be associated with both portfolios and products.
+TagOptions allow administrators to easily manage tags on provisioned products by providing a template for a selection of tags that end users choose from.
+TagOptions are created by specifying a tag key with a set of allowed values and can be associated with both portfolios and products.
 When launching a product, both the TagOptions associated with the product and the containing portfolio are made available.
 
-At the moment, TagOptions can only be disabled in the console.
+At the moment, TagOptions can only be deactivated in the console.
```

Comment:
Should we add this capability to the CDK L2? Seems very easy.

Reply:
We could revisit this, the issue is that you cannot create a tagOption in a disabled state, only on update can it be deactivated.  If we can have way to check for an on update in synth we could handle this.

Reply:
We can just underscore that in the documentation I think.